services/base-service/models/expert/model.js

```diff
@@ -147,10 +147,20 @@ class ExpertModel extends BaseModel {
     // to store totals of works/grants before filtering out any
     let totalWorks = works.length;
     let totalGrants = grants.length;
-    let hiddenWorks = totalWorks - works.filter(w => w.relatedBy?.['is-visible']).length;
-    let hiddenGrants = totalGrants - grants.filter(g =>
-      g.relatedBy && g.relatedBy.some(related => related['is-visible'] && related['inheres_in'])
-    ).length;
+
+    // by default, filter out hidden works/grants if not requested to include them, or if not admin/expert
+    if( options['is-visible'] !== false || !options.admin ) {
+      console.log('filtering out hidden works/grants',doc['@id']);
```

Comment:
@UcDust / @rakunkel-ucd does anyone remember making this change?

Reply:
@qjhart I think I initially added that code to filter out works/grants, but it's further down in the file in the `dev` branch. I don't think I moved it in the file after initially adding it into that `subselect` function.